third_party/xla_client/BUILD

```diff
@@ -77,51 +70,44 @@ cc_library(
-        "@org_tensorflow//tensorflow/compiler/xla/client/lib:svd",
-        "@org_tensorflow//tensorflow/compiler/xla/client/lib:tridiagonal",
-        "@org_tensorflow//tensorflow/compiler/xla/client:global_data",
-        "@org_tensorflow//tensorflow/compiler/xla/client:xla_computation",
-        "@org_tensorflow//tensorflow/compiler/xla/pjrt/distributed",
-        "@org_tensorflow//tensorflow/compiler/xla/pjrt/gpu:se_gpu_pjrt_client",
-        "@org_tensorflow//tensorflow/compiler/xla/pjrt:tpu_client",
-        "@org_tensorflow//tensorflow/compiler/xla/pjrt:pjrt_client",
-        "@org_tensorflow//tensorflow/compiler/xla/pjrt:tfrt_cpu_pjrt_client",
-        "@org_tensorflow//tensorflow/compiler/xla/pjrt:pjrt_c_api_client",
-        "@org_tensorflow//tensorflow/compiler/xla/rpc:grpc_stub",
-        "@org_tensorflow//tensorflow/compiler/xla/service:cpu_plugin",
-        "@org_tensorflow//tensorflow/compiler/xla/service:platform_util",
-        "@org_tensorflow//tensorflow/compiler/xla:statusor",
-        "@org_tensorflow//tensorflow/compiler/xla:xla_data_proto_cc",
-        "@org_tensorflow//tensorflow/compiler/xla/service:hlo_proto_cc",
-        "@org_tensorflow//tensorflow/compiler/xla/hlo/ir:hlo",
-        "@org_tensorflow//tensorflow/compiler/xla/service/spmd:spmd_partitioner",
-        "@org_tensorflow//tensorflow/compiler/xrt:xrt_proto_cc",
-        "@org_tensorflow//tensorflow/compiler/xrt:xrt_server",
-        "@org_tensorflow//tensorflow/compiler/xrt:xrt_utils",
-        "@org_tensorflow//tensorflow/compiler/xrt/cc:xrt_ops",
+        "@xla/xla:debug_options_flags",
+        "@xla/xla:literal",
+        "@xla/xla:literal_util",
+        "@xla/xla:shape_util",
+        "@xla/xla:xla_proto_cc",
+        "@xla/xla/client",
+        "@xla/xla/client/lib:arithmetic",
+        "@xla/xla/client/lib:comparators",
+        "@xla/xla/client/lib:constants",
+        "@xla/xla/client/lib:logdet",
+        "@xla/xla/client/lib:math",
+        "@xla/xla/client/lib:matrix",
+        "@xla/xla/client/lib:pooling",
+        "@xla/xla/client/lib:qr",
+        "@xla/xla/client/lib:slicing",
+        "@xla/xla/client/lib:sorting",
+        "@xla/xla/client/lib:svd",
+        "@xla/xla/client/lib:tridiagonal",
+        "@xla/xla/client:global_data",
+        "@xla/xla/client:xla_computation",
+        "@xla/xla/pjrt/distributed",
+        "@xla/xla/pjrt/gpu:se_gpu_pjrt_client",
+        "@xla/xla/pjrt:tpu_client",
+        "@xla/xla/pjrt:pjrt_client",
+        "@xla/xla/pjrt:tfrt_cpu_pjrt_client",
+        "@xla/xla/pjrt:pjrt_c_api_client",
+        "@xla/xla/rpc:grpc_stub",
+        "@xla/xla/service:cpu_plugin",
+        "@xla/xla/service:platform_util",
+        "@xla/xla:statusor",
+        "@xla/xla:xla_data_proto_cc",
+        "@xla/xla/service:hlo_proto_cc",
+        "@xla/xla/hlo/ir:hlo",
+        "@xla/xla/service/spmd:spmd_partitioner",
         "@org_tensorflow//tensorflow/core:core_cpu",
```

Comment:
alias(
    name = "core_cpu",
    actual = "//tensorflow/core/common_runtime:core_cpu",
    visibility = ["//visibility:public"],
)

Reply:
tf_cuda_library(
    name = "core_cpu",
    hdrs = [
        "composite_device.h",
        "device.h",
        "device_factory.h",
        "device_mgr.h",
        "function.h",
        "function_optimization_registry.h",
        "gradients.h",
        "graph_constructor.h",
        "graph_def_builder_util.h",
        "optimization_registry.h",
        "shape_refiner.h",
        "//tensorflow/core/graph:core_cpu_headers",
        "//tensorflow/core/public:session.h",
        "//tensorflow/core/public:session_options.h",
    ],
    visibility = ["//visibility:public"],
    deps = [
        ":core_cpu_internal",
    ],
)

Reply:
tf_cc_test_mkl(
    name = "mkl_related_tests",
    size = "small",
    srcs = [
        "//tensorflow/core/util:mkl_util_test_srcs",
    ],
    linkstatic = 1,
    deps = [
        ":core",
        ":core_cpu",
        ":core_cpu_internal",

Reply:
# ":core_cpu" - exports the public non-test headers for: graph/: Support for graphs made up of ops common_runtime/: Common code for execution of graphs public/: Public APIs for running graphs

# ":core" - The code for ":core_cpu" plus a GPU runtime